thoth/common/openshift.py

```diff
@@ -1505,6 +1505,7 @@ def schedule_mi_workflow(
         repository: str,
         entities: Optional[str] = None,
         knowledge_path: Optional[str] = None,
+        additional_configuration: Optional[str] = None,
```

Comment:
Are you going to pass parameters such as `-m` to the container?

Reply:
yes, e.g. `"-m"` or `"-mt"` or `"-t"`, possibly few others in future

Reply:
I think this is a good idea, wdyt @fridex ?

Reply:
maybe there should be an enum or something that prevent from passing any string there in this case

Reply:
Or convert those string in parameters that are booleans that activate, deactivate something, so having specific environment variables for each of those letters

Reply:
This was meant just as a quick patch for the other parameters passing. Didn't want to add every one of them, because we still don't know how many of them will be used. Currently there are flags like ` process_knowledge`, `visualize_statistics`, `reviewer_reccomender`, `thoth` and `metrics` and some of them can be removed in future, or potentially changed.

The purpose was to just use passing by string now and in near future change that to explicit parameters. I can make TODO Issues out of this so that it does not get lost through time.

This additional parameters validaiton can be also done in `mi-scheduler` in the end, as the repositories and organizations are also checked there.

Reply:
isn't it enough to have the string assigned in the template for now then? https://github.com/thoth-station/thoth-application/pull/1098#event-4497551515

Reply:
there are basically two workflow types now, the kebechet one and the general mi.
Kebechet uses `"-t"` and the general one will target the `"-m"` option, so therefore I didn't specify anything in the template

Reply:
so what's the status of this @fridex @pacospace ?

Reply:
let's add a TODO if this is stopping you for now and let's plan those adjustments. @fridex thoughts?

Reply:
I've referenced this as a future TODO in `mi-scheduler` here: https://github.com/thoth-station/mi-scheduler/issues/109
therefore marking this as resolved for now

Reply:
I'm not very happy with merging this as it introduces a tech dept. The flags can be mapped to environment variables (with [click, it's just one line](https://click.palletsprojects.com/en/7.x/options/#values-from-environment-variables)) and mapped to parameters supplied to the method on an `OpenShift` instance. Otherwise, the proposed solution is hard to maintain, hard to track the request and hard to extend the solution if the call corresponds to just `openshift.schedule_mi_workflow(additional_paramters="-mt")`. It's prone to errors and does not fit to the rest of the system.

If there is time pressure, let's merge this. But putting more of these things into the application can boil to tech dept we will spend weeks to reduce (I've already been there and it's not something I would like to repeat).

Reply:
alright, parameter changed to be more explicit!